airflow/providers/amazon/aws/operators/eks.py

```diff
@@ -154,11 +204,18 @@ def execute(self, context):
                 subnets=self.resources_vpc_config.get('subnetIds'),
                 nodeRole=self.nodegroup_role_arn,
             )
+        elif self.compute == 'fargate':
+            eks_hook.create_fargate_profile(
+                clusterName=self.cluster_name,
+                fargateProfileName=self.fargate_profile_name,
+                podExecutionRoleArn=self.fargate_pod_execution_role_arn,
+                selectors=self.fargate_selectors,
+            )
```

Comment:
Should we raise an exception on an invalid type?

Reply:
Covered by https://github.com/apache/airflow/pull/18645#discussion_r719868145